examples/tutorials/DeepChemXAlphafold.ipynb

```diff
@@ -0,0 +1,1767 @@
+{
```

Comment:
<p>Let's simplify and omit </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
done, removed the other option of adding custom mmseq files as it wasn't being used in this tutorial